### PR TITLE
Refactor scripts

### PR DESCRIPTION
**Pull request by aju**
I have updated "server" and "watch" scripts to set proper container names for docker. I also added removing of container for "watch" script after hitting CTRL+C (it ). 
I also refactored 'ide-run-scss-lint-from-docker.sh' to get rid of duplicated code.

### CODE REVIEW

File: scripts/server.sh [hunk 1]
```diff
@@ -26,5 +26,5 @@ docker run -t --rm \
 # this will be executed when user hit CTRL+C
 echo
 echo "Please, bear with me while I am stopping and removing container..."
-docker stop brainly-style-guide
-docker rm brainly-style-guide
+docker stop brainly-style-guide-server
```
Comment (mr-mig) [inline scripts/server.sh]: Do we have a non-server container?>

**Comment by vergilius** [comment]
:ok_hand:

**Comment by mr-mig** [comment]
Can you please add some desciption so that the reviewer do not read in the whole sources trying to figure out the context? 
It would be helpful from historical point of view.

File: scripts/ide-run-scss-lint-from-docker.sh [hunk 2]
```diff
@@ -1,9 +1,3 @@
...
-
-# This script will run scss-lint
-
-PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
-
-docker run --rm \
-  -v $PROJECT_DIR:/style-guide \
-  brainly/style-guide bash -c "scss-lint /style-guide/$1 -f JSON | sed 's/\/style-guide\///g'"
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source ${SCRIPT_DIR}/run-scss-lint.sh $1 -f JSON
```
Comment (aju) [inline scripts/ide-run-scss-lint-from-docker.sh]: Remove code duplication use already existing script

File: scripts/run-scss-lint.sh [hunk 3]
```diff
@@ -6,4 +6,4 @@ PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
 
 docker run --rm \
   -v $PROJECT_DIR:/style-guide/ \
-  brainly/style-guide scss-lint $1
+  brainly/style-guide scss-lint "$@"
```
Comment (aju) [inline scripts/run-scss-lint.sh]: proxy any arguments to scss-lint

File: scripts/server.sh [hunk 4]
```diff
@@ -18,7 +18,7 @@ echo "You can now open http://your_docker_ip:8181/dev/docs in web browser"
 
 docker run -t --rm \
   -p 8181:8000 \
-  --name brainly-style-guide \
+  --name brainly-style-guide-server \
```
Comment (aju) [inline scripts/server.sh]: We need to distinct between watch and server

File: scripts/watch.sh [hunk 5]
```diff
@@ -5,8 +5,14 @@
...
+docker run -t --rm --name brainly-style-guide-watch \
   -v $PROJECT_DIR/src:/style-guide/src \
   -v $PROJECT_DIR/dist:/style-guide/dist \
   -v $PROJECT_DIR/gulpfile.js:/style-guide/gulpfile.js \
   brainly/style-guide node_modules/.bin/gulp watch
+
+# this will be executed when user hit CTRL+C
+echo
+echo "Please, bear with me while I am stopping and removing container..."
+docker stop brainly-style-guide-watch
```
Comment (aju) [inline scripts/watch.sh]: Removing container after hitting CTRL+C

**Comment by aju** [comment]
@mr-mig added

**Comment by mr-mig** [comment]
I meant the description for PR, not the code bits :)

Like, what changed and why, like in this guideline: https://github.com/brainly/documents/blob/master/rfc/readme-best-practices.md

**Comment by aju** [comment]
It was refactor so it should not change anything ;) But yes I should have added some detailed description.

**Comment by mr-mig** [comment]
At least now we can easily run 2 containers in parallel :)

File: scripts/ide-run-scss-lint-from-docker.sh (hunk 2 above)
Comment (marcinb) [inline scripts/ide-run-scss-lint-from-docker.sh]: I can't understand why you split it, also naming doesn't make it easier to get as docker-level instructions are actually in `/run-scss-lint.sh` while this file abstracts that away and doesn't even mention word `docker` at all but is still called `ide-run-scss-lint-from-docker` ? :)

Comment (aju) [inline scripts/ide-run-scss-lint-from-docker.sh]: This file is used as scss-lint exec file for WebStorm (and all others form the "family"). I can't pass in config of the plugin options to exec so I created proxy file to do it.  Maybe better name would be `ide-scss-lint-exec.sh` ?

Comment (mr-mig) [inline scripts/ide-run-scss-lint-from-docker.sh]: @marcinb For me it is straightforward - the script runs scss lint from docker (a call to another script is an implementation detail).

**Comment by mr-mig** [comment]
Update the description for PR and merge it.
:+1:

File: scripts/ide-run-scss-lint-from-docker.sh (hunk 2 above)
Comment (marcinb) [inline scripts/ide-run-scss-lint-from-docker.sh]: Ah, I see.

I like the name `ide-scss-lint-exec.sh`, in this case a small comment explaining that `This is an exec script for JetBrains-family IDEs` would be nice to have :+1:

Comment (marcinb) [inline scripts/ide-run-scss-lint-from-docker.sh]: @mr-mig after reading @aju comment I understand the purpose, I don't really use any of these IDEs so it is hard to understand the purpose of this script it is coment-less

**Comment by marcinb** [comment]
Except for the minor thing I mentioned above, :+1:

File: scripts/ide-run-scss-lint-from-docker.sh (hunk 2 above)
Comment (aju) [inline scripts/ide-run-scss-lint-from-docker.sh]: Updated name, added inline comment about it as well. (It is explained in documentation as well) :)

**Comment by aju** [comment]
Thx for review! :) I fixed those things

**Pull request merged**